src/js/components/Carousel.js

```diff
@@ -75,6 +75,20 @@ export default class Carousel extends Component {
     this._unmountHammer();
   }
 
+  _getDOMProps(props) {
```

Comment:
Did you check the Props in utils? I think there might be something similar already there.